z-sgov/z-sgov-slovník.ttl

```diff
@@ -1,19 +1,26 @@
+@prefix :     <https://slovník.gov.cz/základní/slovník#> .
 @prefix owl:  <http://www.w3.org/2002/07/owl#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml:  <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@base         <https://slovník.gov.cz/základní/slovník> .
 
 <https://slovník.gov.cz/základní/slovník>
-    a                                                     owl:Ontology ;
-    owl:imports                                           <https://slovník.gov.cz/základní/glosář/verze/1.0.1>,
-                                                          <https://slovník.gov.cz/základní/model/verze/1.0.0> ;
+    rdf:type                                              owl:Ontology ;
```

Comment:
v turtle platí, že a = rdf:type. Proč to měníš?

Reply:
zjednodušuji, zkracuji, ale nutné to není